torch/_inductor/ir.py

```diff
@@ -3379,14 +3378,6 @@ def process_kernel(cls, kernel, *args, **kwargs):
         _, schemas = get_signature_for_torch_op(kernel, return_schemas=True)
 
         schema = None
-        # For cpp wrapper, when kwargs is not empty, for OpOverloadPacket kernel, we need to
-        # know the exact overload schema to handle the kwargs properly when calling the cpp kernel.
-        if (
-            V.graph.cpp_wrapper
-            and kwargs
-            and isinstance(kernel, torch._ops.OpOverloadPacket)
-        ):
-            schema = try_find_schema(schemas, args, kwargs)
```

Comment:
I remember, this was needed to find the right schema for `torch.repeat_interleave.Tensor`. Do we no longer need this?

Reply:
Right, because Sherlock's https://github.com/pytorch/pytorch/pull/111396/ has made sure we will only see OpOverload at this point.